XML.js

```diff
@@ -1,29 +1,55 @@
-var libxmljs = require("libxmljs");
+'use strict';
+var libxmljs = require('libxmljs');
 
-exports.stringify = stringify;
```

Comment:
JSHint complains about "stringify" and "parse" being called before being defined.